front/src/config/constants.json

```diff
@@ -1,3 +1,7 @@
 {
-  "emailRegex": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$"
+  "emailRegex": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$",
+  "googleOauth": {
```

Comment:
name "constants.json" seems to be not the best option for config values. especially taking that you have "config" folder. 
you can split this "constants.json" into separate parts like "googleauth.json", "email.json" or even ".js" and so on. and you'll be able to include "config/googleauth" for example